bool is_confirmation
string robot_db
int16 tag_id
